script/Arcueid/Day 10/10_00_ARC10_4.txt

```diff
@@ -789,7 +789,7 @@ She doesn't know what blood tastes like...?
 -- 「わたしは吸血種としては半人前だって言ったでしょう。血の味は知らない。それがどんなものなのか、まだわからない。
 "I told you before that I'm a failure as a vampire, right?
  I've never even tasted blood.
- I have no idea what it's like.
+ I have no idea how it's like.
```

Comment:
I think 'what it's like works better here'

Reply:
Agree